corehq/apps/linked_domain/tests/test_release_manager.py

```diff
@@ -148,3 +172,202 @@ def test_app_update_then_fail(self):
             self._assert_release([
                 self._linked_data_view_model(MODEL_APP, detail=AppLinkDetail(app_id=self.master1._id).to_json()),
             ], error="Updated app but did not build or release: Boom!", build_apps=True)
+
+
+class TestReleaseApp(BaseReleaseManagerTest):
+
+    def test_app_not_pushed_if_not_found_with_toggle_disabled(self):
+        unpushed_app = Application.new_app(self.domain, "Not Yet Pushed App")
+        unpushed_app.save()
+        self.addCleanup(unpushed_app.delete)
+        model = self._model_status(MODEL_APP, detail=AppLinkDetail(app_id=unpushed_app._id).to_json())
+        manager = ReleaseManager(self.domain, self.user.username)
+
+        errors = manager._release_app(self.domain_link, model, manager.user)
+
+        self.assertTrue("Could not find app" in errors)
+
+    @flag_enabled('ERM_DEVELOPMENT')
+    def test_app_not_pushed_if_not_found_with_toggle_enabled(self):
```

Comment:
This seems identical to the other test, can't you just call the other test method?

Reply:
Yeah it is identical. I personally don't mind duplication in tests if it communicates a different case. In this case I wanted to differentiate between the feature flag being enabled vs disabled. Either way the app should not be pushed if the app is not found. 

I haven't thought about calling a test method from another test method. So you're suggesting:
```
@flag_enabled('ERM_DEVELOPMENT')
def test_app_not_pushed_if_not_found_with_toggle_enabled(self):
    self.test_app_not_pushed_if_not_found_with_toggle_disabled()
````